pyproj/_list.pyx

```diff
@@ -74,6 +75,12 @@ def get_units_map():
     dict:
         Units supported by PROJ
     """
+    warnings.warn(
+        "The behavior of 'pyproj.get_units_map' is deprecated "
+        "and will change in version 3.0.0.",
```

Comment:
Small comment: it's not very clear what the consequence of this is: what will change? What should the user do instead when seeing this warning?

Reply:
The user really cannot do much here as the change will just happen. The exact change isn't 100% decided, but will definitely be different.

Reply:
Then it is maybe a bit too soon to raise a warning? As this will just be confusing, I might think

Reply:
Well, I wanted to have something. Not sure if there are going to be any more 2.x releases after this next one.